frontend/src/metabase/components/NewItemMenu/NewItemMenu.tsx

```diff
@@ -98,7 +104,10 @@ const NewItemMenu = ({
       },
     );
 
-    if (hasNativeWrite) {
+    if (
```

Comment:
the logic for defining which item to be shown [is defined here](https://www.notion.so/metabase/Tech-Let-embedders-hide-tables-1ae69354c9018057bc1ddcf987443164?pvs=4#1bb69354c90180a3be06ea96c70ecdab).

Basically:
- when embedding: hide Metric and Action
- when `entity_types` is `["table"]` hide Model in addition to Metric and Action.